Readme.md

```diff
@@ -0,0 +1,12 @@
+package pretty
+
+[![GoDoc](https://pkg.go.dev/badge/github.com/kr/pretty)](https://pkg.go.dev/github.com/kr/pretty)
+[![Go Report Card](https://goreportcard.com/badge/github.com/kr/pretty)](https://goreportcard.com/report/github.com/kr/pretty)
+
+    import "github.com/kr/pretty"
+
+    Package pretty provides pretty-printing for Go values.
+
+Documentation
+
+    https://pkg.go.dev/github.com/kr/pretty
```

Comment:
In the plain text version of this readme, GitHub displays this line as a link to the documentation. It would be nice not to lose that link if we are going to make it a markdown file.